src/common/components/layouts/pages/inspect/pdfDocument/PdfDocument.scss

```diff
@@ -6,3 +6,32 @@
 .pdf-page {
   margin-bottom: 7px;
 }
+
+.MuiAlert-root {
```

Comment:
Let's place the alert at the center of the section with document. Please wrap the component that renders the alert and viewer with section and make it `position: relative; overflow: auto;`. And then place the `.MuiAlert-root` at the center with ~10px padding from the top.